markdown_lab/network/async_client.py

```
@@ -0,0 +1,576 @@
+    def __init__(self, config: MarkdownLabConfig):
+        """Initialize async HTTP client with configuration.
+
+        Args:
+            config: Configuration object containing client settings
+        """
+        self.config = config
+        self._session: Optional[httpx.AsyncClient] = None
+        self._semaphore: Optional[asyncio.Semaphore] = None
+
+        logger.debug(
+            f"Initialized AsyncHttpClient with max {config.max_concurrent_requests} concurrent requests"
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)